src/components/AppProvider/README.md

```diff
@@ -481,22 +481,27 @@ To provide access to your initialized Shopify App Bridge instance, we make it av
 import React from 'react';
 import {render} from 'react-dom';
 import * as PropTypes from 'prop-types';
-import {AppProvider} from '@shopify/polaris';
+import {
+  AppProvider,
+  _SECRET_INTERNAL_APP_BRIDGE_CONTEXT,
+} from '@shopify/polaris';
 import {Redirect} from '@shopify/app-bridge/actions';
 
 class MyApp extends React.Component {
   // This line is very important! It tells React to attach the `polaris`
   // object to `this.context` within your component.
-  static contextTypes = {
-    polaris: PropTypes.object,
-  };
+  static contextType = _SECRET_INTERNAL_APP_BRIDGE_CONTEXT;
 
-  redirectToSettings() {
+  componentDidMount() {
     const redirect = Redirect.create(this.context.polaris.appBridge);
```

Comment:
I think this line needs to be updated to:

```suggestion
    const redirect = Redirect.create(this.context);
```
 
https://reactjs.org/docs/context.html#classcontexttype suggests that this.context is set to the value of that particular context when using `contextType` (while the legacy API of `contextTypes` was an object that contained the polaris context, which was an object that contained the appBridge)